docs/permissions/no-self-service-deprecation.md

```diff
@@ -0,0 +1,86 @@
+
+- **No self-service.** Restricts groups from using the query builder to create or edit questions.
+- **Sandbox and block.** Restricts view _and_ query builder access to the underlying data.
+
+Mixing two axes (querying + viewing) to a single permissions setting could yield unexpected behavior. For example, by changing access from "Sandboxed" to "No self-service", an admin might think that they would be _restricting_ that group's access to data. But in that case, the group could potentially see _more_ data, provided the group also had access to collections with existing models, questions, or dashboards.
+
+## What our overhaul of permissions accomplishes
+
+- Splits [view access](./data.md#view-data-permissions) and [query access](./data.md#create-queries-permissions) into two permission dimensions.
+- Makes permissions easier to reason about. A more restrictive permission never gives more access than a less restrictive one.
+
+## Migration table from old permissions to new
+
+This table is just if you're interested in Metabase archeologically. Metabase handles the migration for you.
+
+Before, Metabase had **Data access** and **Native query editing**. Now, Metabase has [View data](./data.md#view-data-permissions) and [Create queries](./data.md#create-queries-permissions). Here's how Metabase migrated each pairing to the new system.
+
+| **Data access**            | **Native query editing** | **->** | **View data**        | **Create queries**            |
+| -------------------------- | ------------------------ | ------ | -------------------- | ----------------------------- |
+| Unrestricted               | Yes                      | **->** | Can view             | Query builder and native code |
+| Unrestricted               | No                       | **->** | Can view             | Query builder                 |
+| No self-service            | No                       | **->** | Can view             | No                            |
+| Blocked                    | No                       | **->** | Blocked              | No                            |
+| Impersonated               | Yes                      | **->** | Impersonated         | Query builder and native code |
+| Impersonated               | No                       | **->** | Impersonated         | Query builder                 |
+| Unrestricted (granular)    | No                       | **->** | Can view             | Query builder (granular)      |
+| Sandboxed (granular)       | No                       | **->** | Sandboxed (granular) | Query builder (granular)      |
+| No self-service (granular) | No                       | **->** | Can view             | No (granular)                 |
+
+## The `No self-service (deprecated)` View access level
+
+If you see the `No self-service (deprecated)` permission setting in **View data** for any group, you should (but don't _need_ to) manually change it at some point.
```

Comment:
I think we should remove the "don't need to" disclaimer.

They should change. In a future release, hopefully very soon, we will remove this permission and move everyone left to Blocked